frontend/src/components/Game.vue

```diff
@@ -84,27 +77,24 @@ export default {
       ev.preventDefault();
       const req = this.gameReq();
       const data = ev.dataTransfer.getData("handIndex");
-      const sentData = { index: data, row: ev.target.getAttribute("rownum") };
-      axios.post(req, sentData);
+      const playData = {
+        index: data,
+        row: ev.target.getAttribute("rownum")
+      };
+      axios.post(req, playData);
     }
   },
   computed: {
     joinLink: function() {
-      const loc = window.location;
-      return (
-        loc.protocol +
-        "//" +
-        loc.hostname +
-        ":" +
-        loc.port +
-        "/#/join-game/" +
-        this.$route.params.gameID
-      );
+      const l = window.location;
```

Comment:
How is `l` better than `loc`?

Reply:
it's literally used only two lines down and I only define the constant to make the code shorter and fit in one line so it's easy to read the format